stream-chat-android-client-sample/src/main/java/io/getstream/chat/android/client/sample/utils/StethoWebSocketsFactory.kt

```diff
@@ -1,82 +0,0 @@
-package io.getstream.chat.android.client.sample.utils
```

Comment:
I'm not very sure about this class... this is not being used and I'm not trying to add new stuff in this PR, just use Flipper instead of Stheto. I decided to remove it.